enhancements/api-review/add-unqualified-search-registries.md

```diff
@@ -46,6 +46,8 @@ Users can currently configure `AllowedRegistries`, `BlockedRegistries`, and `Ins
 
 ### Non-Goals
 
+- Builds and Imagestream Imports will not support the use of short names.
```

Comment:
Could you add justification for why we decided on this?

Reply:
Added